File: public/phase-2-20160620022203/C/another-game-of-life.c
// Name : Another Game of Life
#include <stdio.h>
int main() {
	// input:
	// n :  the size of the map
	// m :  the number of populations that want to
	int n, m, idx, jdx;
	char exists[105]="NO";
	char territory[100][100]={{''}}, territory_tmp[100];
	scanf("%d %d", &n, &m);


	// write your code here
	// store your results in `exists`,`territory`

	// output
	printf("%s", exists);
	if(exists=="YES"){
		for(idx = 0; idx< n; idx++) {
			for(jdx = 0;jdx<n;jdx++) {
				printf("%c ",territory[idx][jdx]);
			}
			printf("\n");
		}
	}
	return 0;
}

File: public/phase-2-20160620022203/JAVA/another-game-of-life.java
// Name : Another Game of Life
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n :  the size of the map
		// m :  the number of populations that want to
		int idx;
		int jdx;
		Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int n = Integer.parseInt(elements[0]);
		int m = Integer.parseInt(elements[1]);


		// write your code here
		// store your results in `exists`,`territory`

		// output
		// Dummy Data
		String exists="NO";
		char[][] territory[100][100]= new char[][] {{''}};

		System.out.println(exists);
		if(exists=="YES"){
			for(idx=0;idx<n;idx++){
				System.out.println(join_string(Arrays.toString(territory[idx])));
			}
		}
	}
	public static String join_string(String inp) {
		return inp.replace("[","").replace("]","").replace(",","");
	}
}